database/migrations/2025_02_04_172448_create_tasks_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('tasks', function (Blueprint $table) {
+            $table->id();
+            $table->string('title');
+            $table->text('description');
+            $table->enum('status', ['pending', 'in_progress', 'completed'])->default('pending');
```

Comment:
```suggestion
            $table->string('status')->default('pending');
```
Here at Light-it, we create enum columns as string columns and define a casting in the model. This is done this way because, given that enums may be subject to change if you create a column as an enum, then it becomes annoying when you need to update the possible values that the column may receive.